### PR TITLE
Create mysql-ssl server in CI

### PR DESCRIPTION
**Pull request by webrouse**
Changes:
- Created separated `mysql-ssl` server in CI with required SSL connection.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -16,95 +16,50 @@ services:
...
+  mysql:
+    image: mysql:${MYSQL_VERSION}
+    command: mysqld --local-infile --port=3306 --default-authentication-plugin=mysql_native_password
     environment:
       MYSQL_DATABASE: test
       MYSQL_ROOT_PASSWORD: rootpassword
 
-  mysql8:
-    image: mysql:8
+  mysql-ssl:
```
Comment (webrouse) [inline docker-compose.yml:39]: Vytvorena sluzba `mysql-ssl`.

File: docker-compose.yml [hunk 2]
```diff
@@ -16,95 +16,50 @@ services:
...
       - ./docker/ssl/certificates:/ssl-cert
       - ssh-keys:/root/.ssh:ro
 
-  mysql5:
-    image: mysql:5.6
-    volumes:
-      - ./docker/ssl/certificates:/ssl-cert
-      - ./docker/mysql/conf.d:/etc/mysql/conf.d
+  mysql:
+    image: mysql:${MYSQL_VERSION}
```
Comment (webrouse) [inline docker-compose.yml:33]: Verzia ako ENV premenna.

File: docker/mysql-ssl/8/conf.d/ssl.cnf [hunk 3]
```diff
@@ -0,0 +1,6 @@
+[mysqld]
+
+ssl-ca = /ssl-cert/ca-cert.pem
+ssl-cert = /ssl-cert/server-cert.pem
+ssl-key = /ssl-cert/server-key.pem
+require_secure_transport = ON
```
Comment (webrouse) [inline docker/mysql-ssl/8/conf.d/ssl.cnf:6]: Vynutenie SSL pre MySQL Server 5.7, ... 5.6 to nepodporuje, ... preto ma kazda verzia svoj config.
Po tejto zmene nie je mozne, aby sa SSL test chcel pripojit bez SSL a preslo by to.
Na skusku staci vypnut SSL v datadir teste `ssl-test-connection` ...

**Review by ondrajodas — APPROVED** [review]
👍

**Pull request merged**